portality/tasks/consumer_main_queue.py

```diff
@@ -24,3 +24,4 @@
 from portality.tasks.async_workflow_notifications import async_workflow_notifications
 from portality.tasks.check_latest_es_backup import scheduled_check_latest_es_backup, check_latest_es_backup
 from portality.tasks.request_es_backup import scheduled_request_es_backup, request_es_backup
+from portality.tasks.harvester import scheduled_harvest
```

Comment:
As we're putting these on the long_running queue, they should be imported in `consumer_long_running.py` instead